.github/docs.yml

```diff
@@ -0,0 +1,55 @@
+name: .NET Docs
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+env:
+  DOTNET_NOLOGO: 1
+  DOTNET_CLI_TELEMETRY_OPTOUT: 1
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+  DOTNET_SYSTEM_GLOBALIZATION_INVARIANT: 1
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 7.0.103
```

Comment:
We use .NET 8, this might break.

Reply:
...can i use preview versions with setup-dotnet?

Reply:
https://github.com/OoLunar/HyperSharp/blob/e9823cc436e11998a598f13b08017c09025631e6/.github/workflows/build-commit.yml#L29-L32